Dockerfile

```
@@ -22,7 +22,7 @@ COPY vendor/ vendor/
 # Build
 RUN CGO_ENABLED=0 GO111MODULE=on go build -mod=vendor -a -o manager main.go
 
-FROM registry.ci.openshift.org/ocp/4.10:base
+FROM debian:buster-slim
```

We use `bullseye` in the first `FROM` so I think this should be
```suggestion
FROM debian:bullseye-slim
```

---

running `apt` will be an issue it prow but we can try only for the testing purpose